src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -15,16 +17,29 @@ internal class ConsoleGui : IDisposable
         private const string ACCEPT_TEXT = "Are you sure you want to select\nthese items to send down the pipeline?";
         private const string CANCEL_TEXT = "Are you sure you want to cancel?\nNothing will be emitted to the pipeline.";
         private const string CLOSE_TEXT = "Are you sure you want to close?";
+        private const string FILTER_LABEL = "Filter";
+        private const string APPLY_LABEL = "Apply";
         private bool _cancelled;
+        private List<string> _itemList;
+        private ListView _listView;
+        private ApplicationData _applicationData;
+        private int[] _listViewColumnWidths;
+        private int _listViewOffset;
+        private Label _filterErrorLabel;
 
         internal HashSet<int> SelectedIndexes { get; private set; } = new HashSet<int>();
         public void Start(ApplicationData applicationData)
         {
             Application.Init();
             var top = Application.Top;
+            _applicationData = applicationData;
+
+            // If we have PassThru, then we want to make them selectable. If we make them selectable,
+            // they have a 8 character addition of a checkbox ("     [ ]") that we have to factor in.
+            _listViewOffset = _applicationData.PassThru ? 8 : 4;
 
             // Creates the top-level window to show
-            var win = new Window(applicationData.Title ?? "Out-ConsoleGridView")
+            var win = new Window(_applicationData.Title ?? "Out-ConsoleGridView")
```

Comment:
I think the title default should be assigned in OutConsoleGridviewCmdletCommand.cs because the class does not know about the cmdlet.